core/trino-main/src/main/java/io/trino/security/AccessControlUtil.java

```diff
@@ -67,47 +59,4 @@ public static void checkCanKillQueryOwnedBy(Identity identity, Identity queryOwn
         }
         accessControl.checkCanKillQueryOwnedBy(identity, queryOwner);
     }
-
-    private static class FullIdentityEquality
```

Comment:
let's squash "Remove obsolete AccessControlUtil.FullIdentityEquality" with the change to `Identity.equals`

Reply:
Squashed.